web3.js/src/connection.ts

```diff
@@ -4278,6 +4278,24 @@ export class Connection {
     }
   }
 
+  /**
+   * get the stake minimum delegation
+   */
+  async getStakeMinimumDelegation(
+    commitment?: Commitment,
```

Comment:
Close! I really did mean to take a config object here, even if for now it only contains a single property. This makes the method call future proof; if we need to add another property then we have a place to do it. There are other examples in this file of calls that take a config. Check it out!

Optional config, optional `commitment` property inside.